src/utils/validateName.ts

```diff
@@ -36,43 +32,63 @@ export function isInvalid(fields: IFormField[]) {
   value = +value;
-  if (isNaN(value)) return "CPU must be a valid number";
-  if (+value.toFixed(0) !== value) return "CPU cores must be a valid integer";
-  if (value < 1) return "Minimum allowed CPU cores is 1";
-  if (value > 32) return "Maximum allowed CPU cores is 32";
+  if (isNaN(value)) return "CPU must be a valid number.";
+  if (+value.toFixed(0) !== value) return "CPU cores must be a valid integer.";
+  if (value < 1) return "Minimum allowed CPU cores is 1.";
+  if (value > 32) return "Maximum allowed CPU cores is 32.";
 }
 
 export function validatePortNumber(value: string): string | void {
   if (value === "") return null;
   let port = +value;
-  if (isNaN(port)) return "Port must be a valid number";
-  if (port.toFixed(0) !== value) return "Port must be a valid integer";
-  if (port < 1) return "Minimum allowed port is 1";
-  if (port > 65535) return "Maximum allowed port is 65535";
+  if (isNaN(port)) return "Port must be a valid number.";
+  if (port.toFixed(0) !== value) return "Port must be a valid integer.";
+  if (port < 1) return "Minimum allowed port is 1.";
+  if (port > 65535) return "Maximum allowed port is 65535.";
 }
 
 export function validatePreCode(value: string): string | void {
-  if (value === "") return "Presearch registration code is required";
-  if (!PRECODE_REGEX.test(value)) return "Invalid presearch registration code";
-  if (value.length !== 32) return "Presearch registration code must be 32 characters long";
+  if (value === "") return "Presearch registration code is required.";
+  if (!PRECODE_REGEX.test(value)) return "Invalid presearch registration code.";
+  if (value.length !== 32) return "Presearch registration code must be 32 characters long.";
+}
+
+export function validateProfileName(name: string): string | void {
+  if (name.length < 2) return "Name must be at least 2 characters.";
+  if (!ALPHA_ONLY_REGEX.test(name[0])) return "Name can't start with a number, a non-alphanumeric character or a whitespace.";
+  if (!PROFILE_NAME_REGEX.test(name)) return "Name can only include alphanumeric characters and spaces.";
+  if (name.length > 15) return "Name must be at most 15 characters.";
 }
 
 export function validatePassword(value: string): string | void {
   if (value.length < 6) return "Password must be at least 6 characters";
-}
\ No newline at end of file
+  if (value.length > 15) return "Password must be at least 15 characters";
+}
+
+export function validateToken(token: string): string | void {
+  if (token.length < 6) return "Token must be at least 6 characters";
+  if (!ALPHA_NUMS_ONLY_REGEX.test(token)) return "Token can't contain any characters other than alphabets and numbers.";
+  if (token.length > 15) return "Token must be at most 15 characters";
+}
+
+export function validateIP(IP: string): string | void {
+  if (IP.length < 6) return "IP must be at least 6 characters";
+  if (!IP_REGEX.test(IP)) return "IP can't contain any characters other than alphabets and numbers.";
```

Comment:
Can we modify the error msg to be `Invalid IP range`
also, the regex should cover the min/max validation